ci/sha256map-regenerate/default.nix

```
@@ -0,0 +1,11 @@
+{ writeScript
+, runtimeShell
+, lib
+, python37
+, nix-prefetch-git
+}:
+writeScript "sha256map-regenerate.sh" ''
+  #!${runtimeShell}
+  export PATH="${lib.makeBinPath [ nix-prefetch-git python37 ]}"
+  ${python37}/bin/python ${./sha256map-regenerate.py}
+''
```

Here's one way I like to do PATH wrapping with scripts (shell or python):

```suggestion
{ stdenv
, lib
, python37
, nix-prefetch-git
}:
stdenv.mkDerivation {
  name = "sha256map-regenerate";
  src = ./sha256map-regenerate.py;
  dontUnpack = true;
  dontConfigure = true;
  dontBuild = true;
  installPhase = ''
    bin="$out/bin/${name}"
    sed $src \
      '2i import sys; ${lib.concatMapStringsSep ";" (p: "sys.path.insert(0, \"${p}/bin\")") [ nix-prefetch-git python37 ]}' \
      "$bin"
    chmod 755 "$bin"
    patchShebangs "$bin"
  '';
}
```
(something like it anyway - I haven't tested this example)